arrocke/gloss-translation
# feat: modify and remove language members

## What has changed



* multiselect for modifying a language members roles
* button to remove member from language
* added secondary and tertiary variants for buttons

## Connected Issues



closes #70 

## QA Steps



1. add a member to a language as an admin
2. verify that the user can see the manage language view and can't translate
3. now add the translator role
4. verify that the user can translate
5. now remove the admin role
6. verify that the user can't access manage language view
7. now remove the member
8. verify that the user can only read the translation view and can't translate

## Post-Deployment
